task/grype/0.1/grype.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: grype
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, grype
+    tekton.dev/displayName: "grype"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    A vulnerability scanner for container images and filesystems.
+    Easily install the binary to try it out. 
+    Works with Syft, the powerful SBOM (software bill of materials) tool for container images and filesystems.
+  workspaces:
+    - name: source-dir
+      optional: true
+  params:
+    - name: ARGS
+      description: The Arguments to be passed to Grype command.
+      type: array
+    - name: GRYPE_IMAGE
+      default: docker.io/anchore/grype:0.31.1@sha256:f1a4547b34d16a0eee53c1adf4e411456d09d10c5d885fd0b94df30df1cf81b9
```

Comment:
I prefer this format of `IMAGE:TAG@sha256:BLAH`, since you get the best of having a tag and a digest, however old versions of podman and maybe other runtimes don't support both tag and digest.

I just want to check we're happy with this. If we can't have this format it's at least a good idea to do `IMAGE@sha256:BLAH # tag` so you (allegedly) know what tag the digest refers to

Reply:
that's super cool, thanks.